inst/extdata/arriba_example.tsv

```diff
@@ -1,4 +1,4 @@
-gene1	gene2	strand1.gene.fusion.	strand2.gene.fusion.	breakpoint1	breakpoint2	site1	site2	type	direction1	direction2	split_reads1	split_reads2	discordant_mates	coverage1	coverage2	confidence	closest_genomic_breakpoint1	closest_genomic_breakpoint2	filters	fusion_transcript	reading_frame	peptide_sequence	read_identifiers	tumor_id	gene1--gene2	annots
+#gene1	gene2	strand1(gene/fusion)	strand2(gene/fusion)	breakpoint1	breakpoint2	site1	site2	type	direction1	direction2	split_reads1	split_reads2	discordant_mates	coverage1	coverage2	confidence	closest_genomic_breakpoint1	closest_genomic_breakpoint2	filters	fusion_transcript	reading_frame	peptide_sequence	read_identifiers	tumor_id	gene1--gene2	annots
```

Comment:
As in the other example file: do we always have a # sign?

Reply:
yes the original outputs from the callers have "#" We had them formatted to remove the "#" in file we are using for OpenPBTA but since we want the users to use the output file directly from the callers I've changed them to default caller ouptut